src/components/ContextMenu.scss

```diff
@@ -73,4 +73,12 @@
   .context-menu-option:focus {
     z-index: 1;
   }
+
+  @media screen and (max-width: 1000px) {
+    .context-menu-option {
+      div:nth-child(2) {
```

Comment:
instead of the nth-child selector lets add a class `context-menu-option__shortcut` in https://github.com/excalidraw/excalidraw/blob/e9cb7ee77cea33f44731b605a78d1999a9d5f733/src/components/ContextMenu.tsx#L55 and apply the css to this class. This way it will be clear readability wise and also if the structure changes it wont break

Reply:
@ReneCode  @ad1992  Perfect , I will change these both changes  in next pr Thanks !!! 🙇🏼